### PR TITLE
Always log basic data for all grabs

### PR DESCRIPTION
**Pull request by cdzombak**
The XSSH grabber isn't outputting anything to the log, which confuses zsearch (it interprets an empty log file as a failure). The easiest way to fix that is for us to always output log info.

This should fix the apparent SSH scan errors we're seeing now.

### CODE REVIEW

**Comment by cdzombak** [comment]
This output looks like:

```
Aug 28 13:26:06.103 [INFO] log: Started grab at 2017-08-28T13:26:06-04:00
Aug 28 13:26:06.106 [INFO] log: Finished grab (0 success; 0 failure) at 2017-08-28T13:26:06-04:00
```

File: main.go [hunk 1]
```diff
@@ -359,9 +359,15 @@ func main() {
 	decoder := zlib.NewGrabTargetDecoder(inputFile, config.LookupDomain)
 	marshaler := zlib.NewGrabMarshaler()
 	worker := zlib.NewGrabWorker(&config)
+
 	start := time.Now()
+	zlog.Infof("Started grab at %s", start.Format(time.RFC3339))
```
Comment (zakird) [inline main.go]: lowercase please!

File: main.go [hunk 2]
```diff
@@ -359,9 +359,15 @@ func main() {
 	decoder := zlib.NewGrabTargetDecoder(inputFile, config.LookupDomain)
 	marshaler := zlib.NewGrabMarshaler()
 	worker := zlib.NewGrabWorker(&config)
+
```
Comment (ajholland) [inline main.go]: Can you remove these newlines? Unless you did them purposefully for readability.

File: main.go (hunk 1 above)
Comment (cdzombak) [reply main.go]: Addressed in f23d962

File: main.go (hunk 2 above)
Comment (cdzombak) [reply main.go]: I did that purposefully for readability, but if we don't like them I can revert them.

File: main.go (hunk 1 above)
Comment (cdzombak) [reply main.go]: (and typo fixed in 9e1d4ad)

**Comment by cdzombak** [comment]
Travis is failing with

```
$ go get ./...
package github.com/zmap/zlint/zlint: cannot find package "github.com/zmap/zlint/zlint" in any of:
	/home/travis/.gimme/versions/go1.8.1.linux.amd64/src/github.com/zmap/zlint/zlint (from $GOROOT)
	/home/travis/gopath/src/github.com/zmap/zlint/zlint (from $GOPATH)
The command "go get ./..." failed and exited with 1 during .
```

**Comment by ajholland** [comment]
@cdzombak That's a side effect of zcertificate importing zlint which just had a structure change. I'm pretty sure we don't use zcertificate and could be removed?

File: main.go (hunk 2 above)
Comment (ajholland) [reply main.go]: I'll defer to zakir, I'm not a large fan though. ¯\_(ツ)_/¯

Comment (zakird) [reply main.go]: This is all going away in like a month. Not a fan, but it just does not matter right now.

Comment (ajholland) [reply main.go]: :+1:

**Pull request merged**